WordPress/Classes/ViewRelated/NUX/SiteInfoHeaderView.swift

```diff
@@ -0,0 +1,30 @@
+import UIKit
+
+/// A subclass of BlogDetailHeaderView styled for use in the login flow.
+///
+class SiteInfoHeaderView: BlogDetailHeaderView {
+
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+
+        configureStyles()
+    }
+
+
+    func configureStyles() {
+
+        titleLabel.font = WPFontManager.systemSemiBoldFont(ofSize: 15.0)
```

Comment:
👍.  Probably we should go ahead and update WPStyleGuide+Blog.configureCellForLogin as well.